docker/ci/setup.sh

```diff
@@ -62,6 +62,7 @@ EOF
         # Keep synchronized with `cargo install --version N.N.N cargo-pgx` in Readme.md and Cargo.toml
         cargo install cargo-pgx --version =0.2.4 --root ~/pgx/0.2
         cargo install cargo-pgx --version =0.4.5 --root ~/pgx/0.4
+        cargo install cargo-pgx --version =0.5.0 --root ~/pgx/0.5
```

Comment:
We're no longer using 2 and 4, right?

Let's drop the other two and also drop the `--root`.

Or is something stopping us from doing that?

Reply:
It looks like we don't use 0.2/0.4 anywhere, removed.